### PR TITLE
Small components

### PR DESCRIPTION
**Pull request by JadeZYX**
1 complete functionality for adding an answer button, so that once click button, a form will be pop up on page, and user can have the interaction like upload photos, fill out some inputs and submit form.
Please review AnswerModal this component.
2 user gets access to report an answer and up for a helpful answer. 
    user gets access to up for a helpful question. 
Please review HelpfulNess this component.

### CODE REVIEW

**Comment by JadeZYX** [comment]
[Display username and time for answer](https://trello.com/c/5DH0rUWm/113-display-username-and-time-for-answer)

**Comment by JadeZYX** [comment]
[Report link for answer](https://trello.com/c/U5Z0C1Jf/114-report-link-for-answer)

**Comment by JadeZYX** [comment]
[Add answer button for question](https://trello.com/c/ZsWJeDij/112-add-answer-button-for-question)

File: client/src/components/QuestionAnswerPart/AnswerModal.jsx [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+      }
+      Promise.all(promisesUpload)
+        .then(() => { setPhotosUrl(url); });
+    } else {
+      alert('No More Than 5 Photos PLEASE');
+    }
+  };
+    // console.log('show photosUrl state array: ', photosUrl);
+
+  const IsvalidEmail = function (value) {
```
Comment (corneliusrenken) [inline client/src/components/QuestionAnswerPart/AnswerModal.jsx:68]: should maybe be isValidEmail?

**Review by corneliusrenken — APPROVED** [review]
Looks good Jade! Left a comment but it's being very nit-picky. 

I'm not sure how much functionality you still have to implement, but be sure to not forget tests! We can look at testing together on monday as well if you want 👍

**Pull request merged**